zebra/zebra_rib.c

```diff
@@ -2665,6 +2665,9 @@ void rib_unlink(struct route_node *rn, struct route_entry *re)
 
 	nexthops_free(re->fib_ng.nexthop);
 
+	if (re->opaque)
```

Comment:
testing for NULL is not needed for XFREE

Reply:
fixed